luminaire/optimization/hyperparameter_optimization.py

```diff
@@ -243,11 +243,10 @@ def _objective_part(self, data, smoothed_series, args):
                     labels = labels + anomaly_flags_list
                     probs = probs + anomaly_probabilities_list
 
-            weights = ((1 - np.array(labels)) + 1) / float(len(labels))
-            if args[3]['model'] == 'LADStructuralModel' and mdape:
-                cost = (0.5 * mdape) + (0.5 * log_loss(labels, probs, sample_weight=weights))
+            if args[3]['model'] == 'LADStructuralModel' and mape:
+                cost = (mape * log_loss(labels, probs)) ** 0.5
```

Comment:
Any reason we're not using `sqrt` instead? I believe it's usually a good amount faster since it's specialized, and also easier to read.

Reply:
No specific reason. I can change it if it is fast. Although, it's an operation on scalars rather than over arrays.

Reply:
True, it's run per-series so performance isn't really perceptable unless you run millions of them. I like sqrt better for the reability though :)